pkg/testutils/testutils.go

```diff
@@ -274,6 +274,20 @@ func CleanupTestNamespaces() {
 	cleanupNamespaces(nses...)
 }
 
+// CleanupCRDIfExists checks whether the eetests.e2e.hnc.x-k8s.io CRD is present, if so, delete it.
+func CleanupTestCRDIfExists() {
+	crd := "eetests.e2e.hnc.x-k8s.io"
+	if err := TryRunQuietly("kubectl get crd", crd); err != nil {
+		// crd does not exist, return directly.
+		return
+	}
+	// undo this before deleting the crd, otherwise webhook will deny the request.
+	MustRun("kubectl hns config delete-type --group e2e.hnc.x-k8s.io --resource eetests")
```

Comment:
Does this work for all the other tests where the CRD doesn't exist? If not, maybe change it to `TryRun`.

Reply:
If CRD does not exist, this function will return early.

Reply:
sg